app/brave_generated_resources.grd

```diff
@@ -346,6 +346,17 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         Check details
       </message>
 
+      <!-- Adblock contenxt menu -->
+      <message name="IDS_ADBLOCK_CONTEXT_TOOLS" desc="The text label of adblock submenu in the context menu (Brave Extension for historical reasons)">
+        Brave Extension
```

Comment:
I'm confused by this - isn't it currently just `Brave`?

![20240903_13h32m06s_grim](https://github.com/user-attachments/assets/5646ef39-ea21-4c7b-94bd-b3ab186a57a9)

Reply:
Fixed.